hack/build-jsonnet.sh

```diff
@@ -37,7 +37,7 @@ for file in "${files[@]}"; do
 	}&
 
 	# wait for at least one of the jobs to finish if there are more than maxProc jobs
-	while [[ $(jobs -r | wc -l ) -ge "$maxProc" ]]; do wait -n; done
+	while [[ $(jobs -r | wc -l ) -ge "$maxProc" ]]; do wait; done
```

Comment:
why this change?

Reply:
ok `-n` is invalid option for mac terminal so I needed to remove that while running generate command, accidentally went in change :)

Reply:
could you revert then?